requirements.txt

```diff
@@ -3,12 +3,11 @@
 #     platform-specific requirements (e.g. sys.platform == 'win32')
 
 # Avoid 1.6 and 1.7 due to method=restore bug: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4209#issuecomment-1739644328
-# dipy 1.8 and above cause conflicts, see https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4319#issuecomment-1864670675
-dipy<1.6
+dipy!=1.6.*,!=1.7.*,<1.9.0
 # PyTorch's Linux distribution is very large due to its GPU support,
 # but we only need that for training models. Our users only need CPU.
 --extra-index-url https://download.pytorch.org/whl/cpu # append_to_freeze
-ivadomed
+ivadomed @ git+https://github.com/ivadomed/ivadomed@master # append_to_freeze
```

Comment:
Note that the failing code quality check is due to the link above, which isn't actually a link that can be accessed via `curl`.

Reply:
I'm thinking `requirements.txt` and `requirements-freeze.txt` can be excluded from the link checker like `CHANGES.md`, since their non-comment links get tested by the installation checks anyway.

Reply:
Maybe there's still some value in checking the links found in the comments of `requirements.txt`? (e.g. in case an context-providing link for a version pin becomes inaccessible.) 

But, given that this is a minor corner case, I think I agree that we could be excluding the requirements files.